_layouts/default.html

```diff
@@ -49,9 +49,7 @@
             </a>
           {% endif %}
 
-          {% if site.github.is_project_page %}
-            <p class="repo-owner"><a href="{{ site.github.repository_url }}">{{ site.github.repository_name }}</a> is maintained by <a href="{{ site.github.owner_url }}">{{ site.github.owner_name }}</a>.</p>
-          {% endif %}
+          {% include sidebar.html %}
```

Comment:
Thanks for the tip. Just what I was looking for.

Though it is the first time I [choose a theme](https://help.github.com/articles/creating-a-github-pages-site-with-the-jekyll-theme-chooser/#using-the-jekyll-theme-chooser-with-an-existing-repository), I guess Github prefers to keep the current content as sidebar footer or header and have and empty default `_includes/sidebar.html`. Also add a note to the README, reading it, this change might still violate the projects philosophy: 'simplicity rather than flexibility'.

Reply:
> Also add a note to the README, reading it, this change might still violate the projects philosophy: 'simplicity rather than flexibility'.

I agree with @jo-pol. When we originally created the theme, we chose not to use includes to keep things as simple as possible for new users.